### PR TITLE
Add a function to port BI VA loadouts to ACE Arsenal

### PR DESCRIPTION
**Pull request by alganthe**
**When merged this pull request will:**
- Adds a function that can be called to port over BI VA loadouts to ACE Arsenal.

### CODE REVIEW

File: addons/arsenal/functions/fnc_portVALoadouts.sqf [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/*
+ * Author: alganthe
+ * Port VA loadouts to ACE Arsenal
```
Comment (jonpas) [inline addons/arsenal/functions/fnc_portVALoadouts.sqf]: Missing final dot.

File: addons/arsenal/functions/fnc_portVALoadouts.sqf [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ *
+ * Arguments:
+ * 0: Write over similarly named loadouts in ACE Arsenal <BOOL>
```
Comment (jonpas) [inline addons/arsenal/functions/fnc_portVALoadouts.sqf]: `<BOOL> (default: false)`

**Comment by jonpas** [comment]
Maybe add a button somewhere?

**Comment by alganthe** [comment]
it need to load the loadout before saving it, that could be easily abused.

**Comment by bux** [comment]
Force it to only work in SP

**Comment by alganthe** [comment]
now that I think about it I could add a button in 3DEN instead of changing the ACE Arsenal interface.

**Comment by alganthe** [comment]
If I add such a button, should I place it under tools or make an ACE category for the main bar?

**Comment by alganthe** [comment]
I removed the override argument (it always does it now) and added a button under tools in 3DEN:
https://imgur.com/a/wWsfn

**Comment by alganthe** [comment]
added an error message for both:
- No VA loadouts to import
- No player unit

should be good to go.

File: addons/arsenal/stringtable.xml [hunk 3]
```diff
@@ -433,5 +433,17 @@
             <Italian>Impossibile aprire l'arsenale ACE</Italian>
             <Japanese>ACE 武器庫を開けません</Japanese>
         </Key>
+        <Key ID= "STR_ACE_Arsenal_portLoadoutsText">
+            <English>import BI VA loadouts to ACE Arsenal</English>
```
Comment (jonpas) [inline addons/arsenal/stringtable.xml]: `Import` (capitalization).

I think something like `Import Vanilla Arsenal loadouts to ACE Arsenal` would be more understandable, but longer.

File: addons/arsenal/stringtable.xml [hunk 4]
```diff
@@ -433,5 +433,17 @@
...
+            <French>Importe les loadouts de BI VA dans ACE Arsenal</French>
+        </Key>
+        <Key ID= "STR_ACE_Arsenal_portLoadoutsPlayerError">
+            <English>No player unit available!</English>
```
Comment (jonpas) [inline addons/arsenal/stringtable.xml]: I would add instruction on what to do to make it work, such as `No player unit available! Place a unit and mark it as "Player"` or something.

File: addons/arsenal/stringtable.xml [hunk 5]
```diff
@@ -433,5 +433,17 @@
...
+            <French>Aucune unité joueur disponible !</French>
+        </Key>
+        <Key ID= "STR_ACE_Arsenal_portLoadoutsLoadoutError">
+            <English>No loadouts to import</English>
```
Comment (jonpas) [inline addons/arsenal/stringtable.xml]: No `!` or `.` here?

File: addons/arsenal/stringtable.xml (hunk 3 above)
Comment (alganthe) [reply addons/arsenal/stringtable.xml]: The issue is that it translates poorly into other languages, on the other hand "BI VA" is pretty much universal.

File: addons/arsenal/stringtable.xml (hunk 4 above)
Comment (alganthe) [reply addons/arsenal/stringtable.xml]: huh, surprisingly you can fit a lot more characters than I thought in those messages

File: addons/arsenal/stringtable.xml [hunk 6]
```diff
@@ -434,16 +434,16 @@
...
         <Key ID= "STR_ACE_Arsenal_portLoadoutsText">
-            <English>import BI VA loadouts to ACE Arsenal</English>
+            <English>Import BI VA loadouts to ACE Arsenal</English>
             <French>Importe les loadouts de BI VA dans ACE Arsenal</French>
         </Key>
         <Key ID= "STR_ACE_Arsenal_portLoadoutsPlayerError">
-            <English>No player unit available!</English>
-            <French>Aucune unité joueur disponible !</French>
+            <English>No player unit available! Place a unit and mark it as "Player".</English>
+            <French>Aucune unité joueur disponible ! Placez une unité et marquez la en tant que "joueur"</French>
```
Comment (jonpas) [inline addons/arsenal/stringtable.xml]: Missed final dot? ^^

Comment (alganthe) [reply addons/arsenal/stringtable.xml]: dammit !

**Pull request merged**